src/runServer.ts

```diff
@@ -25,9 +26,9 @@ if (maxRetries) config = Object.assign({}, config, { maxRetries });
 if (propagationDelayWaitMillis) config = Object.assign({}, config, { propagationDelayWaitMillis });
 const retries: number = parseInt(maxSystemRetries || "1", 10);
 
-function start(retries: number, config: any, databaseDir: any) {
```

Comment:
Better type for isWarpSync here? (although i see they were already any before you modified this line)

Reply:
Need to process into a boolean as well, since it's a string being covertly passed to something expecting a boolean.

Reply:
fixed processing IS_WARP_SYNC env variable, typed the other parameters.